From 3838f68b46b37050749a066dd5e956ca930b5658 Mon Sep 17 00:00:00 2001
From: Dongdong Liu <liudongdong3@huawei.com>
Date: Fri, 30 Jun 2023 15:37:35 +0800
Subject: doc: fix kernel patch link in hns3 guide

[ upstream commit c6a4242b86a020b79d48f9dcc533be113a8df7c8 ]

The LSC support of VF driver depends on a patch link in kernel PF
driver.
But current the link is invalid, so fixes it.

Add a blank line after the link.

Fixes: 80006b598730 ("doc: add link status event requirements in hns3 guide")
Cc: stable@dpdk.org

Signed-off-by: Dongdong Liu <liudongdong3@huawei.com>
---
 doc/guides/nics/hns3.rst | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/guides/nics/hns3.rst b/doc/guides/nics/hns3.rst
index 5373ec5a8f..82fa79dd73 100644
--- a/doc/guides/nics/hns3.rst
+++ b/doc/guides/nics/hns3.rst
@@ -54,7 +54,8 @@ Firmware 1.8.0.0 and later versions support reporting link changes to the PF.
 Therefore, to use the LSC for the PF driver, ensure that the firmware version
 also supports reporting link changes.
 If the VF driver needs to support LSC, special patch must be added:
-`<https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/drivers/net/ethernet/hisilicon/hns3?h=next-20210428&id=18b6e31f8bf4ac7af7b057228f38a5a530378e4e>`_.
+`<https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=18b6e31f8bf4ac7af7b057228f38a5a530378e4e>`_.
+
 Note: The patch has been uploaded to 5.13 of the Linux kernel mainline.
 
 
-- 
2.23.0

